tests/utilities.py

```diff
@@ -0,0 +1,24 @@
+"""
+Assorted utilities useful for the tests.
+"""
+
+import os
+import contextlib
+
+try:
+    import urllib.request as urllib_request  # for python 3
+except ImportError:
+    import urllib2 as urllib_request  # for python 2
+
+import pytest
+import glob
+
+from .get_remote_data import get_datafile
```

Comment:
we don't want to put this in xarray_subset_grid.

I imagine we will put something like this in libgoods, or maybe gridded, but not here.